lib/stm32/common/dma_common_f24.c

```diff
@@ -775,6 +776,20 @@ void dma_set_memory_address_1(uint32_t dma, uint8_t stream, uint32_t address)
 }
 
 /*---------------------------------------------------------------------------*/
+/** @brief DMA Stream Get the Transfer Block Size
+
+@param[in] dma unsigned int32. DMA controller base address: DMA1 or DMA2
+@param[in] stream unsigned int8. Stream number: @ref dma_st_number
+@returns unsigned int16. Number of remaining data words to transfer (65535
+maximum).
+*/
```

Comment:
There's no need for the [in] and the "unsigned int32" stuff.  Doxygen figures that out itself from the code.  Try and use the full * on each line like you should see in most other comments.

You can even drop the "@brief" and simply put a "." (full stop) at the end of the first line if you like.